CoinMarketCap/DataContracts/CryptocurrencyMarketPairs.cs

```diff
@@ -0,0 +1,270 @@
+﻿using System.Collections.Generic;
+using System.Runtime.Serialization;
+
+
+namespace CoinMarketCap.DataContracts
+{
+
+    [DataContract]
+    public class CryptocurrencyMarketPairs
+    {
+        /// <summary>
+        /// The CoinMarketCap ID for this cryptocurrency.
+        /// </summary>
+        [DataMember(Name = "id")]
+        public int Id { get; set; }
+
+        /// <summary>
+        /// The name of this cryptocurrency.
+        /// </summary>
+        [DataMember(Name = "name")]
+        public string Name { get; set; }
+
+        /// <summary>
+        /// The symbol for this cryptocurrency.
+        /// </summary>
+        [DataMember(Name = "symbol")]
+        public string Symbol { get; set; }
+
+        /// <summary>
+        /// The number of active market pairs listed for this cryptocurrency. This number is filtered down to only matching markets if a matched parameter is used.
+        /// </summary>
+        [DataMember(Name = "num_market_pairs")]
+        public int Num_Market_Pairs { get; set; }

```

Comment:
This should be named in Pascal case, `NumMarketPairs`, rather than with underscores.